ui/pages/notifications/notification-components/lido-stake-ready-to-be-withdrawn/lido-stake-ready-to-be-withdrawn.tsx

```diff
@@ -95,6 +96,17 @@ export const components: NotificationComponent<LidoReadyWithDrawnNotification> =
       ),
       body: {
         type: 'body_onchain_notification',
+        Account: ({ notification }) => {
+          if (!notification.address) {
+            return null;
+          }
```

Comment:
Good! Since we cannot guarantee that the address will be there for older notifications